birdhouse/components/README.rst

```diff
@@ -142,44 +142,44 @@ There are 2 tests that need to be performed:
 
 * Can autodeploy deploy the PR from ``master`` branch, the stable reference point?
 
-  * This could fail if some changes in the PR are incompatible with autodeploy. For example: ``./pavics-compose.sh`` calls some binaries that do not exist in the autodeploy docker image.
+  * This could fail if some changes in the PR are incompatible with autodeploy. For example: ``birdhouse compose`` calls some binaries that do not exist in the autodeploy docker image.
 
 * Can autodeploy be triggered again successfully, after the PR is live?
 
   * This could fail if the PR renamed some files and forgot to add the old file names to a ``.gitignore`` file.  Then old file names will appear as new uncommitted files and autodeploy will halt because it expects a clean working directory.
 
 Here is a sample setup to test autodeploy:
 
-* Have 2 checkout directories.  One is for starting the stack using ``./pavics-compose.sh``, the other one is to push new bogus changes to trigger the autodeploy mechanism.
+* Have 2 checkout directories.  One is for starting the stack using ``birdhouse compose``, the other one is to push new bogus changes to trigger the autodeploy mechanism.
 
 .. code-block:: shell
 
-  # this one for running pavics-compose.sh
+  # this one for running birdhouse compose
   git clone git@github.com:bird-house/birdhouse-deploy.git birdhouse-deploy
 
   # this one for triggering autodeploy
   git clone git@github.com:bird-house/birdhouse-deploy.git birdhouse-deploy-trigger
 
-* Set ``AUTODEPLOY_PLATFORM_FREQUENCY`` in ``env.local`` to a very frequent value so you do not have to wait too long for autodeploy to trigger.
+* Set ``BIRDHOUSE_AUTODEPLOY_PLATFORM_FREQUENCY`` in ``env.local`` to a very frequent value so you do not have to wait too long for autodeploy to trigger.
 
 .. code-block:: shell
 
   # go to the main checkout
   cd birdhouse-deploy/birdhouse
 
   # ensure the scheduler component is enabled, otherwise autodeploy will not work
-  echo 'export EXTRA_CONF_DIRS="$EXTRA_CONF_DIRS ./components/scheduler" >> env.local
+  echo 'export BIRDHOUSE_EXTRA_CONF_DIRS="$BIRDHOUSE_EXTRA_CONF_DIRS ./components/scheduler" >> env.local
 
-  # set AUTODEPLOY_PLATFORM_FREQUENCY
+  # set BIRDHOUSE_AUTODEPLOY_PLATFORM_FREQUENCY
   # can set to more frequent than 5 minutes if your machine is capable enough
-  echo 'export AUTODEPLOY_PLATFORM_FREQUENCY="@every 5m"' >> env.local
+  echo 'export BIRDHOUSE_AUTODEPLOY_PLATFORM_FREQUENCY="@every 5m"' >> env.local
 
   # if scheduler container already running:
-  # recreate scheduler container for new AUTODEPLOY_PLATFORM_FREQUENCY to be effective
-  ./pavics-compose.sh stop scheduler && ./pavics-compose.sh rm -vf scheduler && ./pavics-compose.sh up -d
+  # recreate scheduler container for new BIRDHOUSE_AUTODEPLOY_PLATFORM_FREQUENCY to be effective
+  birdhouse compose stop scheduler && birdhouse compose rm -vf scheduler && birdhouse compose up -d
```

Comment:
Not automatically since we don't have any sort of installation script that would do that. We recommend adding it to your path here:

https://github.com/bird-house/birdhouse-deploy/blob/2c3140f46545bef99fbfee34794098e177e6d9e5/birdhouse/README.rst?plain=1#L45

Reply:
Ah right, sorry read the code change too fast yesterday.